### ISSUE TITLE
Feature Request: multiple channels

### ISSUE THREAD

**Issue by hedin-hiervard** [jenkinsci/discord-notifier-plugin#8]
It's currently not possible to notify 2 channels and to add the plugin to the post-build step twice (it's greyed out). I think it's possible from a Pipeline, but I don't have a Pipeline job. Maybe as a workaround, someone could provide a simple downstream Pipeline job that only takes the upstream job info and sends it to Discord?

**Comment by KocproZ** [comment]
Jenkins doesn't allow to add post build action more than once, and it can't be changed.
I think that using pipeline is the only way to do this.

**Comment by hedin-hiervard** [comment]
> I think that using pipeline is the only way to do this.

Can you please help me to create a pipeline job that is run after the main job and the only thing it does - sends the messages to the several Discord channels with build summary and artifact links?

**Comment by melt7777** [comment]
If you do not want to make pipeline jobs, 

You could use the post-build action "Build other projects" to have it call downstream freestyle or multi-configuration projects that have the Discord notifier as post-build actions.

**Comment by hedin-hiervard** [comment]
> If you do not want to make pipeline jobs,
> 
> You could use the post-build action "Build other projects" to have it call downstream freestyle or multi-configuration projects that have the Discord notifier as post-build actions.

Yeah, but will it have access to the artifacts of the upstream project?

**Comment by melt7777** [comment]
Ah, I didn't think of that. Still learning! :)

**Comment by KocproZ** [comment]
Sadly I can't help you with creating such pipeline as I don't know how to do it :/

**Comment by melt7777** [comment]
> 
> 
> > If you do not want to make pipeline jobs,
> > You could use the post-build action "Build other projects" to have it call downstream freestyle or multi-configuration projects that have the Discord notifier as post-build actions.
> 
> Yeah, but will it have access to the artifacts of the upstream project?

You're right. I've explored a bit and it looks like you're going to need to do a pipeline for this one. The Journey into learning Groovy and making Pipelines stored in Github or other version control is a beautiful thing and makes life so much better. It's worth checking out. 

Simply create a new pipeline and use the following snippet inside it. Note: I use the declarative pipeline syntax.

Pipeline you can do something like 

```groovy
node ('master') {
    stage ('Notify') {
def discordDesc = "Description here.."
(...)
discordSend description: discordDesc, 
                  footer: discordFooter, 
                  link: env.JOB_URL, 
                  result: currentBuild.currentResult, 
                  title: discordTitle, 
                  webhookURL: discordURL, 
                  successful: currentBuild.resultIsBetterOrEqualTo('SUCCESS'),
                  thumbnail: discordImage,
                  notes: "Here's some Notes! @<id_to_mention>"
   }
}
```

in the middle of your states you just have things like this discordSend example
https://github.com/jenkinsci/discord-notifier-plugin/issues/10#issuecomment-523563933

**Comment by bkostrowiecki** [comment]
Maybe is it a good idea to have multiple `Webhook URL` fields in `Discord Notifier` post build action?

**Comment by Andre601** [comment]
> Maybe is it a good idea to have multiple `Webhook URL` fields in `Discord Notifier` post build action?

Either that or allow usage of a separator like a comma, semicolon or (white)space in the field to input multiple.
I myself was hoping for a solution that isn't using a pipeline, so I hope this can be looked into.
